Description: Debian modifications to the configuration file.
Author: David Gil <dgil@telefonica.net>
--- a/base_conf.php.dist
+++ b/base_conf.php.dist
@@ -47,7 +47,7 @@
  set this to /base
 
  */
-$BASE_urlpath = '';
+$BASE_urlpath = '/acidbase';
 
 /* Unique BASE ID.  The below variable, if set, will append its value to the
  * title bar of the browser.  This is for people who manage multiple installs
@@ -77,7 +77,7 @@
  *        $foo = 'c:\tmp'    [OK]
  *        $foo = 'c:\tmp\'   [WRONG]
  */
-$DBlib_path = '';
+$DBlib_path = '/usr/share/php/adodb';
 
 
 /* The type of underlying alert database
@@ -87,7 +87,6 @@
  *  MS SQL Server : 'mssql'
  *  Oracle      : 'oci8'
  */
-$DBtype = 'mysql';
 
 /* Alert DB connection parameters
  *   - $alert_dbname   : MySQL database name of Snort alert DB
@@ -99,11 +98,15 @@
  *  This information can be gleaned from the Snort database
  *  output plugin configuration.
  */
-$alert_dbname   = 'snort_log';
-$alert_host     = 'localhost';
-$alert_port     = '';
-$alert_user     = 'snort';
-$alert_password = 'mypassword';
+
+###### Begin of variables configured through dbconfig-common
+
+require('/etc/acidbase/database.php');
+# Adjust dbconfig-common names
+if ($DBtype =='pgsql') 
+    $DBtype = 'postgres';
+
+###### End of variables configured through dbconfig-common
 
 /* Archive DB connection parameters */
 $archive_exists   = 0; # Set this to 1 if you have an archive DB
@@ -487,7 +490,7 @@
 /*
  The below line should not be changed!
  */
-$BASE_path = dirname(__FILE__);
+$BASE_path = '/usr/share/acidbase';
 
 // _BASE_INC is a variable set to prevent direct access to certain include files....
 define( '_BASE_INC', 1 );
